Library/Homebrew/build.rb

```diff
@@ -105,7 +105,8 @@ def install
 
     formula.extend(Debrew::Formula) if ARGV.debug?
 
-    formula.brew do
+    formula.brew do |f, stg|
```

Comment:
I personally prefer `stage_context`, though `staging_context` sounds more correct to my non-native understanding of the English language.

> Don't think spelled-out `stage` can be used because a `stage` method might be called inside these.

A variable will always take precedence over a same-named method. There's no method calling going on in `|<stuff>|` because that's basically an argument list (like in method definitions).